blosc/frame.c

```diff
@@ -1024,13 +980,13 @@ int frame_update_header(blosc2_frame* frame, blosc2_schunk* schunk, bool new) {
   }
 
   FILE* fp = NULL;
-  if (frame->sdata == NULL) {
+  if (frame->framebuf == NULL) {
```

Comment:
Why not use `buffer` instead of `framebuf`?

Reply:
I was hesitating and decided for `framebuf`.  I think `framebuf` is more explicit about the contents of the buffer, but I don't have a strong opinion on this.

Reply:
I ended using `cframe`, which is more meaningful IMO.